lib/Runtime/Library/JavascriptSimdBool16x8.h

```diff
@@ -10,7 +10,7 @@ namespace Js
     {
     private:
         DEFINE_VTABLE_CTOR(JavascriptSIMDBool16x8, JavascriptSIMDType);
-
+        static const char16 _typeName[];
```

Comment:
Please use PascalCase for constants as per coding convention [https://github.com/Microsoft/ChakraCore/wiki/Coding-Convention.](https://github.com/Microsoft/ChakraCore/wiki/Coding-Convention.) I am not sure why there is _ here.